package.json

```diff
@@ -142,10 +141,10 @@
     "ts-node": "^2.0.0",
     "tslint": "~4.3.1",
     "typedoc": "^0.5.3",
-    "typescript": "2.0.10",
+    "typescript": "~2.1.5",
```

Comment:
@pglazkov This is really interesting... I wonder why they say it's not working.

I did some [crazy things](https://github.com/angular/angular/pull/13294#issuecomment-274891832) with typescript nightly with no issues... AOT 100%

I wonder if it's `ngc-webpack` to "blame"